app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,49 @@
+class QuestionsController < ApplicationController
+  before_action :set_question, only: [:show, :update, :destroy]
+  skip_before_action :authenticate_user
+  def index
+    questions = Question.all
+    render json: questions, status: 200
+  end
+
+  def show
+      render json: @question, status: 200
+  end
+
+  def create
+    question = Question.new(questions_params)
+    if question.save
+      render json: question, status: :created
+    else
+      render json:  question.errors, status: 400
+    end
+  end
+
+  def update
+    if @question.update(questions_params)
```

Comment:
```ruby
if @question.update(question_params)
render json: @question, status: 200
else
render json: @question.errors, status: :unprocessable_entity
end
```
Handle the `conditions`, else the frontend may assume that the request was successful.